packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js

```diff
@@ -76,7 +78,14 @@ export class MyAccountSignIn extends PureComponent {
                   validateOn={ ['onChange'] }
                   validationRule={ {
                       isRequired: true,
-                      inputType: VALIDATION_INPUT_TYPE.password
+                      inputType: VALIDATION_INPUT_TYPE.password,
+                      showErrorAsLabel: true,
+                      customErrorMessages: {
+                          onRangeFailMin: `Password should be at least ${minimumPasswordLength} characters`
```

Comment:
Please, use a translatable string here + think of a way to reuse password fields + validation logic (now there is too much duplicated code)